qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +66,39 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
+        obs = self.pm.importable_formats
+        exp = {
+            'IntSequenceDirectoryFormat':
+                FormatRecord(format=IntSequenceDirectoryFormat,
+                             plugin=self.plugin),
+            'MappingDirectoryFormat':
+                FormatRecord(format=MappingDirectoryFormat,
+                             plugin=self.plugin),
+            'IntSequenceV2DirectoryFormat':
+                FormatRecord(format=IntSequenceV2DirectoryFormat,
+                             plugin=self.plugin),
+            'IntSequenceFormatV2':
+                FormatRecord(format=IntSequenceFormatV2,
+                             plugin=self.plugin),
+            'FourIntsDirectoryFormat':
+                FormatRecord(format=FourIntsDirectoryFormat,
+                             plugin=self.plugin),
+            'IntSequenceFormat':
+                FormatRecord(format=IntSequenceFormat,
+                             plugin=self.plugin)
+        }
+        self.assertEqual(obs, exp)
+
+    def test_importable_formats_with_dummy(self):
```

Comment:
ping on [renaming the test](https://github.com/qiime2/qiime2/pull/275#discussion_r120426185)